CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift

```diff
@@ -29,6 +29,7 @@ final class CameraPane: UIView {
             light: UIColor.GiniCapture.dark1,
             dark: UIColor.GiniCapture.dark1).uiColor().withAlphaComponent(0.4)
         captureButton.setTitle("", for: .normal)
+        captureButton.accessibilityIdentifier = "CaptureButton"
```

Comment:
```suggestion
        captureButton.accessibilityIdentifier = "CaptureButtonIdentifier"
```
@SyedaGini  I remember you will check what the the proper style for the manual identifier.

Reply:
Do we want to decide this now in this pr? I have some suggestions for this but I think it will be to much for now when we haven't decided that we will continue this approach.

Reply:
At least investigate how we plan to do it and document it, does it sound good for you @SyedaGini @ValentinaIancu-Gini ?

Reply:
yes yes we should investigate if anyone provides some guidelines